src/redprl/refiner.fun

```diff
@@ -17,7 +17,7 @@ struct
 
   structure Hyp =
   struct
-    fun Project z alpha jdg =
+    fun Project z _ jdg =
```

Comment:
@favonia That's a good idea!

For now, let's replace `alpha` with `_` and we can think of helper functions in a future PR. Ultimately, it may be helpful to have some kind of wrapper around rules anyway.